app/forms/steps/address/lookup_form.rb

```diff
@@ -2,17 +2,17 @@ module Steps
   module Address
     class LookupForm < Steps::BaseFormObject
       attribute :postcode, :string
-      validates :postcode, presence: true, uk_postcode: true, unless: :clear_address?
+      validates :postcode, presence: true, uk_postcode: true
 
       private
 
       def persist!
-        return true unless clear_address? || changed?
+        return true unless changed?
 
         record.update(
           # The following are dependent attributes that need to be reset
           attributes_to_reset.merge(
-            postcode: (postcode unless clear_address?)
+            postcode:
```

Comment:
Just trying to get my head around this. Wondering if now there is nothing to reset on this page

Reply:
Yeah good point, i don't think there is...

Reply:
We can clean this up later when we have time to think it through fully